### ISSUE TITLE
Change the value of max_binlog_cache_size

### ISSUE THREAD

**Issue by hh-project-config** [monasca/monasca-docker#523]
Thanks for providing this useful container. 

I am doing a research aiming at finding issues in configuration files. I have a question about one MySQL config: `max_binlog_cache_size`. It seems the [official document](https://dev.mysql.com/doc/refman/8.0/en/replication-options-binary-log.html#sysvar_max_binlog_cache_size) says "The maximum recommended value is 4GB; this is due to the fact that MySQL currently cannot work with binary log positions greater than 4GB."

However, the default value is 18446744073709551615, which is much larger than the recommended value. 

Shall we change the value to 4GB? 
Thanks!

**Comment by witekest** [comment]
Sounds sensible but shouldn't the upstream default be updated in the first place?

**Comment by hh-project-config** [comment]
Yes, I agree. They actually have acknowledged the bug but haven't fixed it https://bugs.mysql.com/bug.php?id=94487. Maybe it takes a long time for the official to make even a small change.
